### PR TITLE
Update dockerhub to use images in quay.io

### PR DESCRIPTION
**Pull request by xiuwang**
Docker Hub rate limiting policies，so change to use quay.io images
@openshift/devexp-qe Please help review, thanks

### CODE REVIEW

File: features/registry/registry.feature [hunk 1]
```diff
@@ -59,10 +59,10 @@ Feature: Testing registry
     Given I have a project
     Given docker config for default image registry is stored to the :dockercfg_file clipboard
     Then I run the :image_mirror client command with:
-      | source_image | docker.io/library/busybox:latest=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage1:v1 |
-      | dest_image   | centos/ruby-25-centos7:latest=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage2:v1    |
-      | a            | <%= cb.dockercfg_file %>                                                                     |
-      | insecure     | true                                                                                         |
+      | source_image | quay.io/openshifttest/busybox:latest=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage1:v1        |
+      | dest_image   | quay.io/openshifttest/hello-openshift:aosqe=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage2:v1 |
```
Comment (liangxia) [inline features/registry/registry.feature:63]: Is the dest_image update to quay expected ? @xiuwang

Comment (xiuwang) [reply features/registry/registry.feature:63]: @liangxia yes, it's expected. dest_image is a placeholder we added.
oc image mirror <source_image> <dest_image>

Comment (liangxia) [reply features/registry/registry.feature:63]: Ack

**Pull request merged**